src/ec/packages/ec-component-hero-banner/hero-banner.story.js

```diff
@@ -15,105 +16,97 @@ import heroBanner from './ecl-hero-banner.html.twig';
 import notes from './README.md';
 
 const PrepareBanner = data => {
+  getExtraKnobs(data);
+  if (data.link.link.label) {
+    data.link.link.label = text(
+      'Link label',
+      data.link.link.label,
+      buttonLabels.required
+    );
+  }
+
+  if (data.title) {
+    data.title = text('Title', data.title, buttonLabels.required);
+  }
+
+  if (data.title) {
+    data.title = text('Title', data.title, buttonLabels.required);
+  }
+
+  if (data.description) {
+    data.description = text(
+      'Description',
+      data.description,
+      buttonLabels.required
+    );
+  }
+
+  if (data.centered) {
+    data.centered = boolean('Centered', data.centered, buttonLabels.optianal);
+  }
+
+  if (data.image) {
+    data.image = text('Image', data.image, buttonLabels.optianal);
+  }
   data.link.icon.path = defaultSprite; // eslint-disable-line no-param-reassign
   return data;
```

Comment:
Would you add also the other knobs for the link? the path, the label..?

Reply:
Still valid, the label is there (with a typo, though: link.Link.label) but the path is not, the icon path is not. We should define a knob for everything that can be set on the component, ideally, it has to be possible to customise the component in the demo with all the different possibilities and then being able to just copy the include code from the notes tab to reproduce the look and feel of the component for how you see it in storybook, that's the goal :)